README.markdown

```diff
@@ -278,8 +278,11 @@ It is possible to specify the useraccount (and password) that the service will r
        <domain>YOURDOMAIN</domain>
        <user>useraccount</user>
        <password>Pa55w0rd</password>
+       <allowserviceaccountlogonright>true</allowserviceaccountlogonright>
```

Comment:
I see "aLLowserviceaccountLogonright" in my browser (Mozilla 35.0.1).
BTW the copy-paste to text editors shows all letters in the lowercase, so seems it's an issue with fonts